packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -89,18 +93,16 @@ function App() {
           setTelemetryProperties(properties);
 
           try {
-            const deviceId = await getUID();
-
-            fetch('https://analytics.strapi.io/track', {
+            fetch('https://analytics.strapi.io/api/v2/track', {
               method: 'POST',
               body: JSON.stringify({
+                // This event is anonymous
                 event: 'didInitializeAdministration',
-                uuid,
+                adminUserId: '',
                 deviceId,
-                properties: {
-                  ...properties,
-                  environment: appInfo.currentEnvironment,
-                },
+                eventPropeties: {},
+                userProperties: { environment: appInfo.currentEnvironment },
```

Comment:
The `environment` should be in the `eventProperties`, it's not tied to that user.